modules/network-observability-loki-install.adoc

```diff
@@ -8,14 +8,13 @@
 
 .Prerequisites
 
-* Installed Loki, recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.6], which allows the ability to specify a `network` tenant for Loki. The `network` tenant uses a specific tenant mode only implemented in version 5.6 of the Loki Operator. 
-* Object storage, such as AWS S3.
+* Supported Log Store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, OpenShift Data Foundation)
 * {product-title} 4.10+.
 * Linux Kernel 4.18+.
 
 //* <Any Loki install prerequisites for using with Network Observability operator?>
 
-There are several ways you can install Loki. One way is by installing the Loki Operator using the {product-title} web console Operator Hub.
+There are several ways you can install Loki. One way is by installing the Loki Operator using the {product-title} web console Operator Hub. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.6], which allows the ability to specify a `network` tenant for Loki. The `network` tenant uses a specific tenant mode only implemented in version 5.6 of the Loki Operator. 
```

Comment:
Instead of having this listed as a prerequisite for Installing Loki, it makes more sense to have the recommendation here. In the Prerequisites, it doesn't make sense to have "Installed Loki" when the user is about to install Loki.